sdk/src/wallet/storage/constants.rs

```diff
@@ -4,16 +4,16 @@
 /// The default storage path.
 pub const DEFAULT_STORAGE_PATH: &str = "./storage";
 
-/// The default RocksDB storage path.
-#[cfg(feature = "jammdb")]
+/// The default jammDB storage path.
+#[cfg(all(not(feature = "rocksdb"), feature = "jammdb"))]
```

Comment:
For completely independent items like this, they should rely only on the `jammdb` feature and be additive.